src/libpriv/rpmostree-scripts.c

```diff
@@ -325,6 +381,7 @@ run_script_in_bwrap_container (int rootfs_fd,
 
   if (!rpmostree_bwrap_run (bwrap, cancellable, error))
     {
+      dump_buffered_output_noerr (pkg_script, &buffered_output);
```

Comment:
Or maybe it's cleaner to restructure the calls here to only call in the `!stdout_is_journal` path?